### PR TITLE
:bug: Update jira instance connection column to use popover

### PR DESCRIPTION
**Pull request by sjd78**
When a jira instance on the main jira configuration table has a connectivity issue, the connection column will show "Not connected". Clicking the text will now open a popover to display additional details about the state of the instance.  The popover replaces a standard modal box with similar information.

Fixes: #855

What it looked like before:
image

What it looks like now:
image

### CODE REVIEW

**Comment by gildub** [comment]
@sjd78, that sounds good, although during a previous conversation with @rromannissen the idea is to display errors from the Jira in code block (same for the tickets errors). 
Be aware the message can't be several lines long too.
So this needs to be validated by UXD.
Thansk

**Comment by gildub** [comment]
@sjd78 , sounds good, I was concerned about the style change but I can see Ramon validation on the related issue.
Could you please rebase ?

**Review by gildub — CHANGES_REQUESTED** [review]
Looks good,
please use I18n for text as in initial code ?

**Comment by sjd78** [comment]
> Looks good, please use I18n for text as in initial code ?

I swapped the `"Connection"` text back to `t("terms.instance")` for the popover title.  I'll leave the description in the body above the code block since the popover design guidelines always include a header and descriptive text.  Having just the header with the code block text looks a bit awkward.

(rebased also)

File: client/src/app/pages/external/jira/components/tracker-status.tsx [hunk 1]
```diff
@@ -26,28 +29,28 @@ const TrackerStatus = ({ connected, message }: ITrackerStatusProps) => {
...
-              variant="link"
-              onClick={() => setCodeModalState(message)}
+            <Popover
+              aria-label="More information about no connection"
+              alertSeverityVariant="danger"
+              headerIcon={<ExclamationCircleIcon />}
+              headerContent={t("composed.error", { what: t("terms.instance") })}
+              bodyContent={
+                <TextContent>
+                  <Text>Jira instance {name} is not connected.</Text>
```
Comment (gildub) [inline client/src/app/pages/external/jira/components/tracker-status.tsx:39]: Can we use i18n text here ?

**Review by gildub — COMMENTED** [review]
So we' re consistent please use i18n whenever there is user facing text.

**Review by ibolton336 — APPROVED** [review]

**Review by gildub — APPROVED** [review]
LGTM, thanks !

**Pull request merged**